src/components/ui/UiStationHeader.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import type { PropType } from 'vue'
+import type { LineNumber } from '@/model/lines.model'
+import IconLine from '@/components/ui/icons/IconLine.vue'
+
+const props = defineProps({
+  line: {
+    type: Number as PropType<LineNumber>,
+    required: true,
+  },
+  nameStation: String,
+  frequency: Number,
+})
+</script>
+<template>
+  <div class="flex flex-col px-0 gap-1 font-dm-sans">
+    <h2 class="text-2xl font-bold -tracking-[1%]">
+      {{ props.nameStation }}
+    </h2>
+    <div class="flex flex-row p-0 gap-2">
+      <IconLine :line="line" :size="'m'"></IconLine>
+      <div
+        class="font-normal not-italic -tracking-[1%] text-base text-neutral-800"
```

Comment:
I guess you can remove the default value like `not-italic text-base font-normal`. CMIIW. Or is there any reason to set it?

Reply:
You're right it's a good point, by removing the default values ​​the text does not change and remains with the correct style